### PR TITLE
[plg_content_pagebreak]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for plg_content_pagebreak
### Testing Instructions

code review

### CODE REVIEW

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c176872df66af5863be22ac3cc2f87dfffb4a6ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10473">issues.joomla.org/joomla-cms/10473</a>.</sub>

File: plugins/content/pagebreak/pagebreak.xml [hunk 1]
```diff
@@ -20,57 +20,84 @@
...
 				>
 					<option value="1">JSHOW</option>
 					<option value="0">JHIDE</option>
 				</field>
-				<field name="article_index_text"  type="text" default=""
+
+				<field
+					name="article_index_text"
+					type="text"
+					default=""
```
Comment (andrepereiradasilva) [inline plugins/content/pagebreak/pagebreak.xml]: move default to after description.

File: plugins/content/pagebreak/pagebreak.xml [hunk 2]
```diff
@@ -20,57 +20,84 @@
...
+					type="list"
 					label="PLG_CONTENT_PAGEBREAK_STYLE_LABEL"
+					description="PLG_CONTENT_PAGEBREAK_STYLE_DESC"
+					default="pages"
 				>
-					<option value="pages">PLG_CONTENT_PAGEBREAK_PAGES</option>
-					<option value="sliders">PLG_CONTENT_PAGEBREAK_SLIDERS</option>
-					<option value="tabs">PLG_CONTENT_PAGEBREAK_TABS</option>
+					<option
+						value="pages"
```
Comment (andrepereiradasilva) [inline plugins/content/pagebreak/pagebreak.xml]: the changes in the `option`s are not needed. Is not a very big line and makes it harder to read IMO.

File: plugins/content/pagebreak/pagebreak.xml [hunk 3]
```diff
@@ -20,57 +20,79 @@
...
+					type="list"
 					label="PLG_CONTENT_PAGEBREAK_STYLE_LABEL"
+					description="PLG_CONTENT_PAGEBREAK_STYLE_DESC"
+					default="pages"
 				>
-					<option value="pages">PLG_CONTENT_PAGEBREAK_PAGES</option>
-					<option value="sliders">PLG_CONTENT_PAGEBREAK_SLIDERS</option>
-					<option value="tabs">PLG_CONTENT_PAGEBREAK_TABS</option>
+					<option value="pages">
+						PLG_CONTENT_PAGEBREAK_PAGES
```
Comment (andrepereiradasilva) [inline plugins/content/pagebreak/pagebreak.xml]: still think the option tags should be in one line leike they were. Is easier to read.

Comment (wojsmol) [inline plugins/content/pagebreak/pagebreak.xml]: See http://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e838b5fa2415ad22808e00c03b4141d6bad2121b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10473">issues.joomla.org/joomla-cms/10473</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on e838b5fa2415ad22808e00c03b4141d6bad2121b<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10473">issues.joomla.org/joomla-cms/10473</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10473">issues.joomla.org/joomla-cms/10473</a>.</sub>

**Pull request merged**